### PR TITLE
Adding small tickers

### PR DESCRIPTION
**Pull request by deanshub**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -106,14 +114,60 @@ export default function Home() {
...
+  const currentYearBalanceAmount = currentYearOccurences.reduce((res, cur) => {
+    return res + cur.amount
+  }, 0)
+  const currentMonthBalanceAmount = currentMonthOccurences.reduce(
+    (res, cur) => {
+      return res + cur.amount
+    },
+    0
+  )
+
```
Comment (shamit6) [inline pages/index.tsx]: please move it to `transactionsCalculator.ts`. i think it makes sense to use `getTransactionsSummeryByPeriod` and not `generateTransactionConfigsOccurances`

File: pages/index.tsx [hunk 2]
```diff
@@ -106,14 +114,60 @@ export default function Home() {
...
+              duration={4}
+            />
+            <Ticker
+              small
+              label="Month AVG"
+              prefix={currentYearBalanceAmount < 0 ? '-' : '+'}
+              number={currentYearBalanceAmount / 12}
+              duration={4}
+            />
+          </div>
```
Comment (shamit6) [inline pages/index.tsx:148]: how it works with next feature of display the chart bu year/month?

File: yarn.lock [hunk 3]
```diff
@@ -2097,11 +2097,16 @@ classnames@2.2.6:
   resolved "https://registry.yarnpkg.com/classnames/-/classnames-2.2.6.tgz#43935bffdd291f326dad0a205309b38d00f650ce"
   integrity sha512-JR/iSQOSt+LQIWwrwEzJ9uk0xfN3mTVYMwt1Ir5mUcSN6pU+V4zQFFaJsclJbPuAUQH+yfWef6tm7l1quW3C8Q==
 
-classnames@^2.2.6, classnames@^2.3.1:
+classnames@^2.2.6:
   version "2.3.1"
   resolved "https://registry.yarnpkg.com/classnames/-/classnames-2.3.1.tgz#dfcfa3891e306ec1dad105d0e88f4417b8535e8e"
   integrity sha512-OlQdbZ7gLfGarSqxesMesDa5uz7KFbID8Kpq/SxIoNGDqY8lSYs0D+hhtBXhcdB3rcbXArFr7vlHheLk1voeNA==
 
+classnames@^2.3.1:
```
Comment (shamit6) [inline yarn.lock:2105]: ?

Comment (deanshub) [reply yarn.lock:2105]: https://github.com/shamit6/get-there/pull/11/files#diff-5ee198ea2b6ebf737659609ae05c0d4652c3aecb605e88cc2bec8d557dcfc9bbR35

File: pages/index.tsx (hunk 2 above)
Comment (deanshub) [reply pages/index.tsx:148]: I don't understand what you're asking

File: pages/index.tsx (hunk 1 above)
Comment (deanshub) [reply pages/index.tsx]: can't really use it since it accepts very different parameters

**Pull request merged**